fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalOlapTableSink.java

```diff
@@ -242,7 +242,9 @@ syncMvWhereClauses, targetTableSlots, groupExpression, getLogicalProperties(),
      */
     @Override
     public PhysicalProperties getRequirePhysicalProperties() {
-        if (targetTable.isPartitionDistributed()) {
+        if (targetTable.isPartitionDistributed()
+                || (targetTable.isPartitionedTable() && targetTable.getPartitions().isEmpty()
+                        && targetTable.getDefaultDistributionInfo().getBucketNum() > 1)) {
```

Comment:
the key point is whether this statement will write more than one tablet? so for auto partition table, we should always set to true?

Reply:
not only the auto partition table, but also zero partition table has this problem